### DISCUSSION TITLE
Welcome to Hacktoberfest2022-python Discussions!

### DISCUSSION THREAD

**Discussion by pravee42** [pravee42/Hacktoberfest2022-python#18] [Announcements]
Use can use this discission for talk about the code and ask doubts spamming allowed

discord server link => https://discord.gg/pSup8ZZV

**Comment by AbbeyIT** [comment]
Is this repository still accepted for the Hacktoberfest 2022? @pravee42

**Reply by pravee42** [reply]
Yes
